Core/Core/InstUI/Views/TextSectionView.swift

```diff
@@ -0,0 +1,70 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public extension InstUI {
+    struct TextSectionView: View {
+        @Environment(\.sizeCategory) private var sizeCategory
+        private let title: String
+        private let description: String
+
+        init(title: String, description: String) {
```

Comment:
Shouldn't it be `public` init? Overall I would prefer if we would use:
```
extension InstUI {
  public struct ...
```
Even though it has the same effect, but to be more selfcontained and explicit.
Same for other public extensions.

Reply:
Yes, the init should be public. We can discuss the public extensions.